decompiler/ff7_field/ff7_field_engine.cpp

```diff
@@ -1600,6 +1880,69 @@ void FF7::FF7CameraInstruction::processInst(Function& func, ValueStack&, Engine*
     }
 }
 
+void FF7::FF7CameraInstruction::processNFADE(CodeGenerator* codeGen)
```

Comment:
It just looked a bit werid to say types ABC go into X, then in X split back into ABC